app/views/expenses_introduction/edit.html.erb

```diff
@@ -1,36 +1,35 @@
 <% content_for :header_title, 'Expenses' %>
-
-<div class='form-card'>
-  <header class='form-card__header'>
-    <div class='step-section-header'>
-      <div class="step-section-header__icon illustration illustration--section-3"></div>
-      <div class='step-section-header__headline'>You're halfway there!</div>
-      <div class='step-section-header__subhead'>
+<div class="form-card">
+  <header class="form-card__header">
+    <div class="step-section-header">
+      <% icon 'section-3' %>
```

Comment:
this `icon` helper does not exist in this codebase, needs to be `<div class="step-section-header__icon illustration illustration--section-3"></div>` per diff